test/dlc_tests/eks/pytorch/training/test_eks_pytorch_training.py

```diff
@@ -171,6 +171,8 @@ def test_eks_pytorch_dgl_single_node_training(pytorch_training, py3_only):
     image_cuda_version = get_cuda_version_from_tag(pytorch_training)
     if Version(image_framework_version) == Version("1.6") and image_cuda_version == "cu110":
         pytest.skip("DGL does not suport CUDA 11 for PyTorch 1.6")
+    if Version(image_framework_version) >= Version("2.0.1") and image_cuda_version == "cu121":
+        pytest.skip("DGL does not support CUDA 12.1 for PyTorch 2.0")
```

Comment:
This skip is okay to leave as-is, because EKS tests don't create resources of their own to run the test. Each test just runs its own pod on a pre-existing cluster.

Just please change the `image_cuda_version` comparision to a `>=`, and use the version comparison approach used at https://github.com/aws/deep-learning-containers/blob/163e4ed6b06dcdb81ec2c10e0065abeb3e69880a/test/sagemaker_tests/pytorch/training/integration/sagemaker/test_smdataparallel.py#L41-L44

Reply:
made change